docs/orchestrating-elastic-stack-applications/elasticsearch/autoscaling.asciidoc

```diff
@@ -0,0 +1,197 @@
+:parent_page_id: elasticsearch-specification
+:page_id: autoscaling
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{parent_page_id}.html#k8s-{page_id}[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Elasticsearch autoscaling
+
+experimental[]
+
+ECK can leverage the autoscaling API introduced in Elasticsearch 7.11 to adjust automatically the number of Pods and the allocated resources in a tier. Currently, autoscaling is supported for Elasticsearch data tiers and machine learning nodes.
```

Comment:
I wonder if we should provide a little more explanation about what is a tier because it's pretty new. Maybe just add this link https://www.elastic.co/guide/en/elasticsearch/reference/current/data-tiers.html?